src/main/webapp/css/buckyless/notifications.less

```diff
@@ -1,140 +1,179 @@
+    }
+    .action {
+      width:10%;
+    }
+  }
 
-// New Stuff Div
+  // New Stuff Div
 
-.header-opaque {
-  width:100%;
-  position:absolute;
-  background-color:rgba(256,256,256,0.8);
-  bottom:0px;
-  z-index:150;
-  div {
-    display:inline-block;
-    float:left;
-    margin:0px 10px;
-    h3 {
-      margin-top:9px;
-      font-size:20px;
+  .header-opaque {
+    width:100%;
+    position:absolute;
+    background-color:rgba(256,256,256,0.8);
+    bottom:0px;
+    z-index:150;
+    div {
+      display:inline-block;
+      float:left;
+      margin:0px 10px;
+      h3 {
+        margin-top:9px;
+        font-size:20px;
+      }
+      p {
+        color:#555;
+        margin:10px 0px;
+      }
     }
-    p {
-      color:#555;
+    .new {
+      text-transform:uppercase;
+      color:#fcf49c;
+      background-color:rgba(183,1,1,0.8);
+      border-radius:8px;
+      padding:3px 8px 1px;
       margin:10px 0px;
+      font-size:12px;
+    }
+    .remove {
+      float:right;
+      padding:10px 0px;
+      color:#555;
+    }
+    .remove:hover {
+      cursor:pointer;
+      color:#333;
     }
   }
-  .new {
-    text-transform:uppercase;
-    color:#fcf49c;
-    background-color:rgba(183,1,1,0.8);
-    border-radius:8px;
-    padding:3px 8px 1px;
-    margin:10px 0px;
-    font-size:12px;
+
+  // News Mockups
+
+  .news-article-title {
+    font-weight:200;
+    position:absolute;
+    bottom:28px;
+    right:15px;
+    color:#333;
+    padding:0;
+    margin:0;
+    font-size:1.8em;
   }
-  .remove {
-    float:right;
-    padding:10px 0px;
-    color:#555;
+  .news-article-description {
+    position:absolute;
+    bottom:8px;
+    right:15px;
+    color:#333;
+    margin:0;
+    padding:0;
   }
-  .remove:hover {
-    cursor:pointer;
+  .news-section-title {
+    font-size:0.7em;
+    position:absolute;
+    bottom:53px;
+    right:5px;
     color:#333;
+    margin:0;
+    padding:0;
+  }
+  .news-article a {
+    color:@color1;
+  }
+  .page-h1 {
+    font-size: 2.3em;
+    color: @color1;
+    padding: 0;
+    font-weight: 200;
   }
-}
 
-// News Mockups
+  /* old styles as backup
 
-.news-article-title {
-  font-weight:200;
-  position:absolute;
-  bottom:28px;
-  right:15px;
-  color:#333;
-  padding:0;
-  margin:0;
-  font-size:1.8em;
-}
-.news-article-description {
-  position:absolute;
-  bottom:8px;
-  right:15px;
-  color:#333;
-  margin:0;
-  padding:0;
-}
-.news-section-title {
-  font-size:0.7em;
-  position:absolute;
-  bottom:53px;
-  right:5px;
-  color:#333;
-  margin:0;
-  padding:0;
-}
-.news-article a {
-  color:@color1;
-}
-.page-h1 {
-  font-size: 2.3em;
-  color: @color1;
-  padding: 0;
-  font-weight: 200;
-}
-
-/* old styles as backup
+  .notifications-list li {
+    font-size: 1.1em;
+    color: #333;
+    background-color: #e7d9c1;
+    border-radius: 4px;
+    margin: 8px 0px;
+    padding: 8px 10px 8px 10px;
+    border:none;
+    border-left: 10px solid @color1;
+    font-weight: 400;
+  } */
```

Comment:
I think we should just delete this. We can always get it from history if we really really really want it :)